### PR TITLE
[v7r3] Fix tornado WMS services

### PR DESCRIPTION
**Pull request by chrisburr**
BEGINRELEASENOTES
Thank you for writing the text to appear in the release notes. It will show up
exactly as it appears between the two bold lines

Please follow the template:
*Subsystem
NEW/CHANGE/FIX: explanation

For examples look into release.notes

ENDRELEASENOTES

### CODE REVIEW

File: src/DIRAC/WorkloadManagementSystem/Service/JobManagerHandler.py [hunk 1]
```diff
@@ -85,7 +85,7 @@ def initializeHandler(cls, serviceInfoDict):
       cls.log.warn("Cannot connect to OptimizationMind!", result['Message'])
     return S_OK()
 
-  def initialize(self):
+  def initializeRequest(self):
```
Comment (chrisburr) [inline src/DIRAC/WorkloadManagementSystem/Service/JobManagerHandler.py:88]: There is still an issue that `self.diracSetup = self.serviceInfoDict['clientSetup']` doesn't exist for tornado.

Perhaps it needs to be added here like the VO is? Else I'm not sure if there was a different plan for this parameter?

https://github.com/DIRACGrid/DIRAC/blob/d8c1c45d087ac543ab99b510470c08a7bd2370ad/src/DIRAC/Core/Tornado/Client/private/TornadoBaseClient.py#L474


Unless I've missed it we should also add the client version to the request as well like was recently done for DISET.

File: src/DIRAC/WorkloadManagementSystem/Service/JobMonitoringHandler.py [hunk 2]
```diff
@@ -70,10 +70,9 @@ def initializeHandler(cls, svcInfoDict):
 
     return S_OK()
 
-  def initialize(self):
+  def initializeRequest(self):
```
Comment (chrisburr) [inline src/DIRAC/WorkloadManagementSystem/Service/JobMonitoringHandler.py:73]: I'm surprised this was passing the integration tests, is it simply that we don't have test coverage for this or is there something wrong with the HTTPS test setup?

**Comment by fstagni** [comment]
I am working on a slightly different fix. What you are doing here for JobMonitoring at least is not correct, IMHO. I will provide the fix later.

File: src/DIRAC/WorkloadManagementSystem/Service/JobMonitoringHandler.py (hunk 2 above)
Comment (fstagni) [reply src/DIRAC/WorkloadManagementSystem/Service/JobMonitoringHandler.py:73]: I want to verify that too, I will try running locally.

**Comment by fstagni** [comment]
If you don't mind I close this in favor of https://github.com/DIRACGrid/DIRAC/pull/5354

**Pull request closed without merging**